engine/dlib/src/dmsdk/dlib/log.h

```diff
@@ -120,23 +117,28 @@ namespace dmLog
  * Log severity
  *
  * @enum
- * @name dmLog::Severity
- * @member dmLog::LOG_SEVERITY_DEBUG
- * @member dmLog::LOG_SEVERITY_USER_DEBUG
- * @member dmLog::LOG_SEVERITY_INFO
- * @member dmLog::LOG_SEVERITY_WARNING
- * @member dmLog::LOG_SEVERITY_ERROR
- * @member dmLog::LOG_SEVERITY_FATAL
+ * @name LogSeverity
+ * @member DM_LOG_SEVERITY_DEBUG
+ * @member DM_LOG_SEVERITY_USER_DEBUG
+ * @member DM_LOG_SEVERITY_INFO
+ * @member DM_LOG_SEVERITY_WARNING
+ * @member DM_LOG_SEVERITY_ERROR
+ * @member DM_LOG_SEVERITY_FATAL
```

Comment:
I wanted to use our logging system from C.
So I needed to put these back into "global scope".
As such I also thought it should have prooper prefixes.
I'm not 100% sure about the logic here, but added "Log" to "Severity" since it belongs to the log header.
Added "DM_" to the enums to avoid name clashes with external headers.
However, the same logic could be said for the "LogSeverity".

Perhaps I should remove the "DM_" prefix for the values 🤔